src/call_builder.ts

```diff
@@ -117,86 +117,85 @@ export class CallBuilder<
 
     const createTimeout = () => {
       timeout = setTimeout(() => {
-        if (es) {
-          es.close();
-        }
-        /* tslint:disable-next-line:no-use-before-declare */
+        console.log("timed out")
```

Comment:
maybe? and maybe a slightly more informative message?
```suggestion
        console.error("timed out")
```

Reply:
lol this was left over from debugging failures in node 16, dropped